internal/pkg/runtime/launcher/options.go

```diff
@@ -141,6 +141,9 @@ type Options struct {
 	// SysContext holds Docker/OCI image handling configuration.
 	// This will be used by a launcher handling OCI images directly.
 	SysContext *types.SystemContext
+
+	// CDIDevices contains the list of CDI device mappings (if any)
+	CDIDevices []string
```

Comment:
As at the CLI layer, lets call this `Device` in case (a) we end up implementing non-CDI `--device` binding, and (b) it moves out of `internal` for 4.0... which is an aim... sensible semver controlled public API.